integration-tests/implementationGuides.test.ts

```diff
@@ -0,0 +1,136 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import { AxiosInstance } from 'axios';
+import waitForExpect from 'wait-for-expect';
+import { Chance } from 'chance';
+import { expectResourceToBePartOfSearchResults, getFhirClient, randomPatient } from './utils';
+import { CapabilityStatement } from './types';
+
+jest.setTimeout(60 * 1000);
+
+describe('Implementation Guides - US Core', () => {
+    let client: AxiosInstance;
+    beforeAll(async () => {
+        client = await getFhirClient();
+    });
+    test('capability statement includes search parameters', async () => {
+        const capabilityStatement: CapabilityStatement = (await client.get('metadata')).data;
+
+        const usCorePatientSearchParams = capabilityStatement.rest[0].resource
+            .filter(x => x.type === 'Patient')
```

Comment:
Nitpick: To make it more readable we can replace `x` with `resource`

Reply:
I agree. Done.